homeassistant/helpers/selector.py

```diff
@@ -153,7 +156,7 @@ class EntityFilterSelectorConfig(TypedDict, total=False):
     integration: str
     domain: str | list[str]
     device_class: str | list[str]
-    supported_features: list[str]
+    supported_features: list[int | str | list[str]]
```

Comment:
The PR description says:

> Allow numeric supported_features for EntityFilterSelectorConfig schema

However, the change also allows single strings, is it a mistake?